File: controllers/ObjectController/@ObjectClass/set_color.m
function set_color(obj, color, alpha)
    obj.color = color;
    obj.alpha = alpha;
    wb_display_set_color(obj.display, color);
    wb_display_fill_oval(obj.display, obj.width/2, obj.height/2, 200, 200);
    wb_display_set_alpha(obj.display, alpha);
end

File: controllers/ObjectController/set_color.py
def getIfromRGB(rgb):
    red = rgb[0]
    green = rgb[1]
    blue = rgb[2]
    RGBint = (red<<16) + (green<<8) + blue
    return RGBint

def set_color(self, color, alpha):
    self.color = color
    
    self.display.setColor(getIfromRGB(color))
    self.display.setAlpha(alpha)
    self.display.fillOval(int(self.display.getWidth()/2), int(self.display.getHeight()/2), 200, 200)